### PR TITLE
[Merged by Bors] - refactor: fix clippy warnings

### PR DESCRIPTION
**Pull request by morenol**
BREAKING CHANGE: `default` method was renamed to `try_default`

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -113,17 +113,17 @@ jobs:
           key: ${{ matrix.os }}-${{ matrix.k8 }}
       - name: Install Minikube for Github runner
         if: startsWith(matrix.k8,'minikube')
-        uses: manusa/actions-setup-minikube@v2.4.3
+        uses: manusa/actions-setup-minikube@v2.7.2
```
Comment (morenol) [inline .github/workflows/ci.yml:116]: changed

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -113,17 +113,17 @@ jobs:
...
         with:
           minikube version: "v1.22.0"
-          kubernetes version: "v1.21.2"
+          kubernetes version: "v1.23.10"
```
Comment (morenol) [inline .github/workflows/ci.yml:119]: changed

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -113,17 +113,17 @@ jobs:
...
           github token: ${{ secrets.GITHUB_TOKEN }}
           driver: docker
       - name: Install k3d
         if: startsWith(matrix.k8,'k3d')
         run: |
-          curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
-          k3d cluster create fluvio-k3d --image rancher/k3s:v1.19.12-k3s1-amd64
+          curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | TAG=${{ env.K3D_VERSION }} bash
```
Comment (morenol) [inline .github/workflows/ci.yml:125]: changed

**Review by nacardin — APPROVED** [review]

**Pull request closed without merging**